Scripts/SL-Histogram.lua

```diff
@@ -92,6 +92,13 @@ local function gen_vertices(player, width, height, desaturation)
 				end
 			end
 		end
+
+		-- Insert a 0 NPS datapoint at the end of the graph, otherwise
+		-- the last measure will not have a nice downwards slope like
+		-- all the other measures but end abruptly at the start of the
+		-- measure.
+		verts[#verts+1] = {{width, 0, 0}, blue}
```

Comment:
Can we only add these vertices if #verts > 0. Seems unnecessary to add otherwise.

Reply:
Yep, I changed it!